redash/utils/sentry.py

```diff
@@ -4,7 +4,7 @@
 from redash import settings, __version__
 
 
-NON_REPORTED_EXCEPTIONS = ['QueryExecutionError']
+NON_REPORTED_EXCEPTIONS = ['BadRequest', 'QueryExecutionError']
```

Comment:
@arikfr nope, only 400s. If you want 404s as well, smash this:

```suggestion
NON_REPORTED_EXCEPTIONS = ['NotFound', 'BadRequest', 'QueryExecutionError']
```

Reply:
Only the author of the PR, can accept suggestions... Do you think we should keep reporting 404s there?

Reply:
The only thing I can think that we should keep 404s for is any manual construction of links in the frontend that might not pop up as a bug during development and testing.

Reply:
Two things about this:

1. In dev/test we don't send Sentry reports anyway.
2. This kind of issues should be tracked from the frontend.